<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>442</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Программа добавления фигур</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="areaHorizontalLayout">
      <item>
       <widget class="QLabel" name="areaLabel">
        <property name="text">
         <string>Площадь фигуры: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numberAreaLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="parametresStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rectanglePage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="widthhorizontalLayout">
          <item>
           <widget class="QLabel" name="widthLabel">
            <property name="text">
             <string>Ширина</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="widthDoubleSpinBox">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="heightHorizontalLayout">
          <item>
           <widget class="QLabel" name="heightLabel">
            <property name="text">
             <string>Высота</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="circlePage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="radiusLabel">
          <property name="text">
           <string>Радиус:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="radiusDoubleSpinBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="typeShapeHorizontalLayout">
      <item>
       <widget class="QLabel" name="typeShapeLabel">
        <property name="text">
         <string>Тип фигуры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeShapeComboBox"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="addShapePushButton">
      <property name="text">
       <string>Добавить фигуру</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QListView" name="shapeListView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>typeShapeComboBox</tabstop>
  <tabstop>widthDoubleSpinBox</tabstop>
  <tabstop>heightDoubleSpinBox</tabstop>
  <tabstop>radiusDoubleSpinBox</tabstop>
  <tabstop>addShapePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
